pages/news/2020-XX-XX-new-in-2-3-track-colors.html

```diff
@@ -35,6 +35,10 @@
 Highlight tracks that you still need to prepare hotcues for or assign a color.
 You can also mark tracks that have bad sound quality and you're planning to replace later.
 
+Of course we didn't forget about controller support:
```

Comment:
Instead of the double negotiation what about
```suggestion
Of course we included controller support, as well:
```